qiime/sdk/visualization.py

```diff
@@ -33,3 +35,14 @@ def _from_data_dir(cls, data_dir, provenance):
         # shutil.copytree doesn't allow the destination directory to exist.
         viz._archiver.save_data(data_dir, distutils.dir_util.copy_tree)
         return viz
+
+    @classmethod
+    def get_index_paths(cls, fp):
```

Comment:
:+1: I think the above lines should be exposed as a public method on the `Archiver` like `get_data_dir()` or similar.